### DISCUSSION TITLE
How to use microcks-cli import with local dev setup that is keycloak less

### DISCUSSION THREAD

**Discussion by nagarro-nikhiljagtiani** [microcks/microcks#638] [Q&A] [answered]
I get an error that keycloak key and secret are required when importing an api spec, but what should be done with keycloak less setup supported in docker compose.

**Comment by lbroudoux** [comment, marked as answer]
Hi! I'm afraid we didn't update the `microcks-cli` since we released the keycloak less setup with docker-compose. This should require an update on the `microcks/microcks-cli` side.

**Comment by lbroudoux** [comment]
I just created an issue to track this one [here](https://github.com/microcks/microcks-cli/issues/23)
